mitodl/bootcamp-ecommerce
# Catalog grid section - #163

#### What are the relevant tickets?
[Ticket](https://trello.com/c/PsnxTAjM/163-home-page-bootcamps-grid-catalog)

#### What's this PR do?
Adds a `CatalogGridPage` section to `HomePage`.

#### How should this be manually tested?
Add the `CatalogGridPage` to `HomePage` and add content to it. Verify the home page displays the content as expected.

#### Screenshots (if appropriate)
Wagtail editing interface
<img width="1482" alt="Screenshot 2020-06-16 at 2 52 00 PM" src="https://user-images.githubusercontent.com/6387678/84761160-78a88080-afe2-11ea-89e1-3e98a601eca5.png">

Mobile layout
![screencapture-bc-odl-local-8099-2020-06-16-14_53_19](https://user-images.githubusercontent.com/6387678/84761182-8100bb80-afe2-11ea-95e0-e3d66fd9ed0d.png)

Tablet layout
![screencapture-bc-odl-local-8099-2020-06-16-14_54_04](https://user-images.githubusercontent.com/6387678/84761192-86f69c80-afe2-11ea-8631-00098ac7604a.png)

Desktop layout
![screencapture-bc-odl-local-8099-2020-06-16-14_54_26](https://user-images.githubusercontent.com/6387678/84761206-8cec7d80-afe2-11ea-9f01-6c3628946e7e.png)
